### PR TITLE
Add new character and weapon information released in Genshin Impact v5.0 Phase 2

### PR DESCRIPTION
**Pull request by sdglitched**
Add new character and weapon information released in Genshin Impact v5.0 Phase 2

- Character: Kinich
Screenshot from 2024-10-28 00-05-57
Screenshot from 2024-10-28 00-06-09

- Weapon: Claymore - Fang of the Mountain King
Screenshot from 2024-10-28 00-07-23
Screenshot from 2024-10-28 00-07-34

Add test for newly added data
```
---------- coverage: platform linux, python 3.12.7-final-0 -----------
Name                                          Stmts   Miss  Cover   Missing
---------------------------------------------------------------------------
...
gi_loadouts/data/char/kinich.py                  17      0   100%
gi_loadouts/data/weap/claymores/fomk.py          12      0   100%
...
---------------------------------------------------------------------------
TOTAL                                         11079      0   100%
Coverage HTML written to dir htmlcov
Coverage XML written to file coverage.xml

=============== 1310 passed, 2 warnings in 115.85s (0:01:55) ===============
  py312-dist: OK (138.30=setup[0.75]+cmd[11.97,4.22,121.36] seconds)
  congratulations :) (138.43 seconds)
```

Fixes: #56 #57

### CODE REVIEW

**Comment by gridhead** [comment]
Works fine on Windows 10 22H2

image

Please make the changes to reflect compatibility with `5.0 Phase 2`.

image

**Comment by sdglitched** [comment]
Modified the the Phase information: `5.0 Phase 2`

Screenshot from 2024-10-28 20-10-01
Screenshot from 2024-10-28 20-10-20

File: assets/back.qrc [hunk 1]
```diff
@@ -41,6 +41,7 @@
     <file>imgs/char/back/kamisato_ayato.webp</file>
     <file>imgs/char/back/kaveh.webp</file>
     <file>imgs/char/back/keqing.webp</file>
+    <file>imgs/char/back/kinich.webp</file>
```
Comment (gridhead) [inline assets/back.qrc:44]: Good work maintaining the same format as the other image assets

File: assets/cons.qrc [hunk 2]
```diff
@@ -40,6 +40,7 @@
     <file>imgs/char/cons/kamisato_ayato.webp</file>
     <file>imgs/char/cons/kaveh.webp</file>
     <file>imgs/char/cons/keqing.webp</file>
+    <file>imgs/char/cons/kinich.webp</file>
```
Comment (gridhead) [inline assets/cons.qrc:43]: Good work maintaining the same format as the other image assets

File: assets/face.qrc [hunk 3]
```diff
@@ -41,6 +41,7 @@
     <file>imgs/char/face/kamisato_ayato.png</file>
     <file>imgs/char/face/kaveh.png</file>
     <file>imgs/char/face/keqing.png</file>
+    <file>imgs/char/face/kinich.png</file>
```
Comment (gridhead) [inline assets/face.qrc:44]: Good work maintaining the same format as the other image assets

File: assets/name.qrc [hunk 4]
```diff
@@ -41,6 +41,7 @@
     <file>imgs/char/name/kamisato_ayato.png</file>
     <file>imgs/char/name/kaveh.png</file>
     <file>imgs/char/name/keqing.png</file>
+    <file>imgs/char/name/kinich.png</file>
```
Comment (gridhead) [inline assets/name.qrc:44]: Good work maintaining the same format as the other image assets

File: assets/side.qrc [hunk 5]
```diff
@@ -41,6 +41,7 @@
     <file>imgs/char/side/kamisato_ayato.png</file>
     <file>imgs/char/side/kaveh.png</file>
     <file>imgs/char/side/keqing.png</file>
+    <file>imgs/char/side/kinich.png</file>
```
Comment (gridhead) [inline assets/side.qrc:44]: Good work maintaining the same format as the other image assets

File: gi_loadouts/__init__.py [hunk 6]
```diff
@@ -4,7 +4,7 @@
 __versdata__ = __metadict__.get("version")
 
 __gicompat_vers__ = "5.0"
-__gicompat_part__ = "1"
+__gicompat_part__ = "2"
```
Comment (gridhead) [inline gi_loadouts/__init__.py:7]: Thanks for taking care of the release phase

**Review by gridhead — APPROVED** [review]
Looks good to me!

Thanks for taking a lead on this! 🎉

**Pull request merged**